scripts/generate_linux_tests.rb

```diff
@@ -71,6 +71,7 @@ def createExtensionFile(fileName, classes)
     file.write "\n"
 
     for classArray in classes
+      file.write "#if swift(<5.6)\n"
```

Comment:
these set of changes seem redundant to me. can you provide more background on what issue these are solving?

Reply:
It's a swiftpm regression, I'm talking with Boris about it